src/diffusers/models/embeddings.py

```diff
@@ -141,6 +156,66 @@ def get_3d_sincos_pos_embed(
     return pos_embed
 
 
+def get_3d_sincos_pos_embed_np(
```

Comment:
make it a private method `_get_3d_sincos_pos_embed_np` and deprecate here too in case anymore wants to use it directly